connector_extension/components/binder.py

```diff
@@ -112,11 +112,17 @@ def dict2id2dict(self, _dict, in_field=True, alt_field=False):
             alt_field=alt_field,
         )
 
-    def dict2id(self, _dict, in_field=True, alt_field=False, unwrap=False):
+    def dict2id(
+        self, _dict, in_field=True, alt_field=False, unwrap=False, raise_on_error=False
```

Comment:
Better put this name:
```suggestion
        self, _dict, in_field=True, alt_field=False, unwrap=False, raise_on_not_found=False
```